chat-sdk-core/src/main/java/co/chatsdk/core/utils/ImageUtils.java

```diff
@@ -35,6 +36,11 @@
 
 public class ImageUtils {
 
+    static {
+        StrictMode.VmPolicy.Builder builder = new StrictMode.VmPolicy.Builder();
+        StrictMode.setVmPolicy(builder.build());
+    }
+
```

Comment:
This allows reading from external storage

Reply:
Are you sure? As far as I'm aware, this code just makes it so that long running processes like network access can't run on the main thread. [See here](https://stackoverflow.com/questions/18100161/strictmode-threadpolicy-builder-purpose-and-advantages) Could you show me where you found that it says that it affects file paths?